src/jsp/com/cubrid/jsp/Server.java

```diff
@@ -53,68 +54,69 @@
 
     private static List<String> jvmArguments = null;
 
-    private ServerSocket serverSocket;
-    private Thread socketListener;
+    private int portNumber = 0;
+    private Thread tcpSocketListener = null;
+    private Thread udsSocketListener = null;
     private AtomicBoolean shutdown;
 
     private static Server serverInstance = null;
 
     private Server(String name, String path, String version, String rPath, String port)
-            throws IOException {
+            throws IOException, ClassNotFoundException {
         serverName = name;
         spPath = path;
         rootPath = rPath;
         shutdown = new AtomicBoolean(false);
 
-        try {
-            int port_number = Integer.parseInt(port);
-            serverSocket = new ServerSocket(port_number);
+        if (OSValidator.IS_UNIX) {
+            String socketName = rPath + "/tmp/junixsocket-" + name + ".sock";
```

Comment:
Please refer to the jsp_get_socket_file_path() function in jsp_comm.c. 
"/tmp" is used only when there is no result of envvar_get ("TMP"), but here "/tmp" is used unconditionally. 
The process of checking the value of envvar_get ("TMP") should be added.

Reply:
I've fixed by passing the envvar_get("TMP") value to the Java SP Server